Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -402,17 +402,15 @@ - (void)loadAdViewAdForParameters:(id<MAAdapterResponseParameters>)parameters ad
     }
     else
     {
-        BannerSize adSize = [self adSizeFromAdFormat: adFormat];
-        
-        self.adView = [[VungleBanner alloc] initWithPlacementId: placementIdentifier size: adSize];
+        VungleAdSize *adSize = [self adSizeFromAdFormat: adFormat
+                                             parameters:parameters];
```

Comment:
max dosent like this kind of extra lines. should be in one line.

Reply:
Updated.